test/extended/router/scoped.go

```diff
@@ -173,15 +173,15 @@ var _ = g.Describe("[sig-network][Feature:Router][apigroup:route.openshift.io]",
 			o.Expect(condition.LastTransitionTime).NotTo(o.BeNil())
 		})
 
-		g.It("should override the route host for overridden domains with a custom value [apigroup:image.openshift.io]", func() {
+		g.It("should override the route host for overridden domains with a custom value", func() {
 
 			routerPod := createOverrideDomainRouterPod(routerImage)
 			g.By("creating a router")
 			ns := oc.KubeFramework().Namespace.Name
 			_, err := oc.AdminKubeClient().CoreV1().Pods(ns).Create(context.Background(), routerPod, metav1.CreateOptions{})
 			o.Expect(err).NotTo(o.HaveOccurred())
 
-			execPod := exutil.CreateExecPodOrFail(oc.AdminKubeClient(), ns, "execpod")
+			execPod := exutil.CreateAgnhostPodOrFail(oc.AdminKubeClient(), ns, "execpod")
```

Comment:
I think these are the only controversial changes, IIUC the ShellImage doesn't work on microshift because there's no image registry in the cluster, and that's where the `tools` image comes from